<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SumoAdd</class>
 <widget class="QMainWindow" name="SumoAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>定期保养模次数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具编号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品料号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MoldID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入模具编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品名称:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>365</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(170, 170, 255);
	border-radius: 15px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(170, 85, 255);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>增加模具</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="LogoLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/images/logo110x30.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="BackGround">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="ProductName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入产品名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LifeNum">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>375</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具设计寿命:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MachineNum">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>195</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>1#</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label7">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>增加机台:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>315</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>穴数:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="KeepNum1">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>255</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大保养模次数:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="KeepNum2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>500000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="KeepNum3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>500000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>315</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>特定易损件管制模次数:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutNum">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProductNum1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剩余订单数1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProductNum2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剩余订单数2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>标准产量:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProductNum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>195</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>标准产量(k/24H)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProductID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>85</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入产品料号</string>
    </property>
   </widget>
   <zorder>BackGround</zorder>
   <zorder>Label8</zorder>
   <zorder>Label3</zorder>
   <zorder>Label2</zorder>
   <zorder>MoldID</zorder>
   <zorder>Label1</zorder>
   <zorder>AddButton</zorder>
   <zorder>LogoLabel</zorder>
   <zorder>ProductName</zorder>
   <zorder>LifeNum</zorder>
   <zorder>Label12</zorder>
   <zorder>MachineNum</zorder>
   <zorder>Label7</zorder>
   <zorder>Label11</zorder>
   <zorder>KeepNum1</zorder>
   <zorder>Label9</zorder>
   <zorder>KeepNum2</zorder>
   <zorder>KeepNum3</zorder>
   <zorder>Label10</zorder>
   <zorder>OutNum</zorder>
   <zorder>ProductNum1</zorder>
   <zorder>Label4</zorder>
   <zorder>ProductNum2</zorder>
   <zorder>Label5</zorder>
   <zorder>Label6</zorder>
   <zorder>ProductNum</zorder>
   <zorder>ProductID</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ProductName</tabstop>
  <tabstop>ProductID</tabstop>
  <tabstop>MoldID</tabstop>
  <tabstop>ProductNum</tabstop>
  <tabstop>ProductNum1</tabstop>
  <tabstop>ProductNum2</tabstop>
  <tabstop>MachineNum</tabstop>
  <tabstop>KeepNum1</tabstop>
  <tabstop>KeepNum2</tabstop>
  <tabstop>KeepNum3</tabstop>
  <tabstop>OutNum</tabstop>
  <tabstop>LifeNum</tabstop>
  <tabstop>AddButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
